release_testing/a2.network.yml

```diff
@@ -0,0 +1,10 @@
+# OBSOLETE since we're not supporting pre-sonic clients anymore
```

Comment:
Rather than comparing opera vs sonic. This scenario can be used to check compatibility between two sonic versions.